packages/shared/src/components/profile/ProfileForm.tsx

```diff
@@ -47,6 +51,7 @@ export default function ProfileForm({
 
   const formRef = useRef<HTMLFormElement>(null);
 
+  const [userTimeZone, setUserTimeZone] = useState<string>(user?.timezone);
```

Comment:
Can you check if there's any way to get the user's timezone so we can select it by default when the `user?.timezone` is null?
This way we can select by default the user's correct timezone

Reply:
Should we select it by default?
This might make bad UX as the user is unaware he might have to save it?

Reply:
Ok, you're right. So in case, it's part of the registration, let's select the local timezone by default. If it's in the edit account details flow we should select the UTC+0 time by default, because this is the actual timezone that is used in case existing users didn't fill it already.